src/mmw/js/src/data_catalog/views.js

```diff
@@ -416,6 +416,25 @@ var StaticResultView = Marionette.ItemView.extend({
                                               .join('; '),
             };
         }
+
+        if (this.options.catalog === 'cinergi') {
+            var categories = _.clone(this.model.get('categories'));
+
+            if (!categories) {
+                return null;
```

Comment:
Should we return `null` or `{ 'top_categories': null }` here?

Reply:
It doesn't seem to make a difference to the template, though in the detail view PR I refactored this and it always returns [`{'top_categories': ??}`](https://github.com/WikiWatershed/model-my-watershed/pull/2429/files#diff-068d5676bbc1115f7893be5af4d3e7d0R422)

Reply:
Since it is refactored in the following PR anyway, let's leave it as is now. Interesting to know that Nunjucks treats `null` and `undefined` as equivalent.